src/main.cpp

```diff
@@ -1,8 +1,7 @@
 /***************************************************************************
  *   Copyright (C) 2008-2013 by Heiko Koehn - KoehnHeiko@googlemail.com    *
- *   Copyright (C) 2013-2014, 2016 by Stephen Lyons                        *
- *                                            - slysven@virginmedia.com    *
- *   Copyright (C) 2014 by Ahmed Charles - acharles@outlook.com            *
+ *   Copyright (C) 2013-2016 by Stephen Lyons - slysven@virginmedia.com    *
```

Comment:
Sometimes I think the FSF likes being pedantic about things that effectively don't matter. I looked at the Boost and Apache documentation on licenses and neither discuss it at all. I was just going for the cleaner aesthetic.

I'll change it back.